m4_sessions/1811_inning/belong_in_tech.md

```diff
@@ -1,30 +1,75 @@
 ## Group Members:
+Justin Stewart
+Jessica Erickson
+Mark Peayer
+Andrew Tobin
 
 
 
 ## Name of Session: 
+Why you belong tech.
 
 ## Session Description 
 
-(Why should students come to this session): Are you interested in biological enhancement? Drop in to learn about biohacking.
+Have you ever felt like an outsider? Is your background not tech related? Come find out why the tech industry really needs you.
 
 ## Session Outcomes 
 
 (What will students learn/leave with?): 
 
-### Recommended Trajectory 
+- Have data and Scientific facts to back diveristy in tech
+- Squash any doubt members belong in tech
+- code switching
+- Being a career changer is ok, new is ok
+
+** Being confident they made the right decision to persue tech.
+** They belong in this community, and nothing disqualifies them.
 
-     Best practices:  No section should be longer than 20 minutes
-     Create a session that YOU would want to attend
-     Share resources in your slack channel
-     All sessions NEED to have a closing activity, reflection, discussion
-     # Introductons 
-     # Intro to new material
-     # Small Group Activity
-     # Closing Discussion
+### Recommended Trajectory 
+     
+ Introduction - and split class into 4 small groups
+ Acticity- 
+      ** possible individual activities with a round robin format
+      Justin - discuss personal fears of why we may not belong and counter them with why we do belong together.
+      Mark - Go over code switching and how it is ok.
+      Jessica - Give factual evidence and scientific data to tool gear up members with hard proof of diversity.
+      Andrew - tbh
+      
+ Closing - Ask for feedback, How can you take this and succeed at turing && beyond.
+ 
 
 ## Session Outline (Agenda for session, include timing for each session and who is facilitating)
 
 
+intro - 
+
+10 mins- 
+*icebreaker* - name, cohort, program, Something you discovered that you are proud of. 
+
+session - 
+
+5 mins-
+mini presentation
+
+count off by 4's and break out into small groups lead by each moderator. 2 minutes
+
+Jessica - Diversity statistics and why they matter
+Mark - code switching, and why its ok. 
+Justin - Your difference does not matter
+Andrew - Positive intelligence ( reference the Book) Silence negative voices.
+
+** each 10 minutes for each round robin^^
+
+fill time for character description
+     did you perspective change with the new descriptions shared by peers
+
+
+closing - 
```

Comment:
What are your closing comments?  What do you want people to take away?  Are people sharing whole group?

Reply:
reflection questions
     * take a minute to think about your life track towards getting you into tech
     * How can your different experiences and skills benefit a team.
     * What did you learn today that you can incorporate into your daily thought process and into the next inning
     * is there something that we could add to this gear up to help get of over mental roadblocks
     * As developer how will you try to make everyone feel welcome and accepted